### DISCUSSION TITLE
suggestions for improvement

### DISCUSSION THREAD

**Discussion by pfreak2000** [mysteriumnetwork/node#5224] [Ideas]
- dark node theme 
- a function that displays the quality and ranking of your node in the dashboard.
- Speed display of own node in dashboard
- the project needs much more advertising, also especially for streamers and downloaders  

Advertising on streaming sites for the project would do wonders. I really wonder why this was not considered earlier.

**Comment by slaygirlz** [comment]
agree for dark mode
